README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
+- Supports browsing and searching apps' music libraries
+- Supports selecting from a list of currently-queued songs, as well as basic back/next control
+- Integrates into the car's audio context, for automatic resume and hardware button control
+- Supports controlling any active music session, even apps that aren't Android Auto compatible
+- Automatically updates the screen to follow the active app
 
-Implemented Features
---------------------
+## Recommended compatible apps:
+
+|Category|Apps|
+|--------|------|
+|🎙 Audiobooks and Podcasts| [![Acast Podcast Player](https://play-lh.googleusercontent.com/QJ8PChzCg_lwdZLnsr_bliT_K88DMNi4UfWmK8zgjF7CgJ5mGzQZFV6ZrJUFiWJFfWS6=s30-rw)](https://play.google.com/store/apps/details?id=com.acast.nativeapp)[![AntennaPod](https://play-lh.googleusercontent.com/VV5fdZWuZolh8Ii8FiuxD7LYCacmT76sCxyuMTm7kyfsHJkBWfkHSZNmqr1UGTo6JZCq=s30-rw)](https://play.google.com/store/apps/details?id=de.danoeh.antennapod)[![The Bob & Tom Show](https://play-lh.googleusercontent.com/bwXFPAGsIBhp8xrFPATUzQk_y0pn230HHbMN9MxJ_GbTt5lN2-Fc_ShzyRtEfSDE9Vk=s30-rw)](https://play.google.com/store/apps/details?id=com.radio.station.BOB.TOM)[![Castbox](https://play-lh.googleusercontent.com/CzFEOgmTdindoUqV4xDpZMknyjplmSUgOU-W9DtTuAZuyzZy_8fn26BZDRXzSwDbOTEh=s30-rw)](https://play.google.com/store/apps/details?id=fm.castbox.audiobook.radio.podcast)[![Google Play Books](https://play-lh.googleusercontent.com/DglqS-eYHQYXnj8M8tmzh3JcKDXcidSo3IzgyCZzci8ZTV9Pmuk8vvIFh9XHOztC3Q=s30-rw)](https://play.google.com/store/apps/details?id=com.google.android.apps.books)[![iVoox](https://play-lh.googleusercontent.com/wt9Ca2lmCCmx-eEC5DwnUBj0VS8hgYkXlymD_Vh6tF8xTvat8xW41F4YBfvnCwLyJ3o=s30-rw)](https://play.google.com/store/apps/details?id=com.ivoox.app)[![Libro.fm Audiobooks](https://play-lh.googleusercontent.com/FAv5D8GukPgAF75mHk-jyCmBalu9c8XlHJnFsDCXx2dXzLfarAhXRu--gPhNYbXwTsw=s30-rw)](https://play.google.com/store/apps/details?id=fm.libro.librofm)[![Listen Audiobook Player](https://play-lh.googleusercontent.com/h1leqRpQBYVx06E_jwzTx25d5p1jJAj7D38y82dX3oLKxUxpBcZ3ybIzS0Cgq8hoZA=s30-rw)](https://play.google.com/store/apps/details?id=com.acmeandroid.listen)[![Player FM](https://play-lh.googleusercontent.com/VfeI7e9DAoVHcsMtAvvvjowaV3B_8NiH1Gc5gpKwld099MsQYefrgdis_uxSSXXHzUA=s30-rw)](https://play.google.com/store/apps/details?id=fm.player)[![Podcast Addict](https://play-lh.googleusercontent.com/m6FeLOkUfP8qTZNXKFSSI8_exI-SlGJRcIArl3gRm3-VninL7l1RdYlPkkf2CfbBnA=s30-rw)](https://play.google.com/store/apps/details?id=com.bambuna.podcastaddict)[![Voice Audiobook Player](https://play-lh.googleusercontent.com/JnpwL-hH58QmiY8J1sQNu-IV9WJyXPDKh_XHQLWEVZSOKAun3aRdXegt6r7NylXkTxQ=s30-rw)](https://play.google.com/store/apps/details?id=de.ph1b.audiobook)|
```

Comment:
I really like this more graphical layout!
It looks like the GFM syntax supports [this format](https://github.github.com/gfm/#example-580) for specifying a title tag, which should allow for hover text. For example, `[![Google Play Books](https://play-lh.googleusercontent.com/DglqS-eYHQYXnj8M8tmzh3JcKDXcidSo3IzgyCZzci8ZTV9Pmuk8vvIFh9XHOztC3Q=s30-rw)](https://play.google.com/store/apps/details?id=com.google.android.apps.books "Google Play Books")` yields [![Google Play Books](https://play-lh.googleusercontent.com/DglqS-eYHQYXnj8M8tmzh3JcKDXcidSo3IzgyCZzci8ZTV9Pmuk8vvIFh9XHOztC3Q=s30-rw)](https://play.google.com/store/apps/details?id=com.google.android.apps.books "Google Play Books").
Can a space be added between each icon, to help separate the more-square ones and to help navigate the list when adding more apps?

Alternatively, can each app somehow be in a separate line in the MD but still be on a single row in the table? It looks like the only way to do it is with a manual `<table>` html syntax... Would converting to RST provide more expressive options?

Reply:
You picked a great size for the icons, they fit perfectly with the text

Reply:
> supports this format for specifying a title tag, which should allow for hover text. 

Ah, didn't know that - it's pretty easy to add them without going one by one :D 

> Can a space be added between each icon, to help separate the more-square ones and to help navigate the list when adding more apps?

Sure!

> Alternatively, can each app somehow be in a separate line in the MD but still be on a single row in the table? 

I tried playing around with it to do that but didn't quite manage to get it working 🤔

> Would converting to RST provide more expressive options?

I suspect you're referring to [reStructured Text](https://gist.github.com/javiertejero/4585196) ? I haven't worked with it, but I can take a look. Worst case scenario we can extract the app listing in a separate Markdown file and import it somehow if that's possible ? Or an `iframe` for that matter. 😅

Reply:
RST has a clever [substitution](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions) method, which helps clean up the syntax for the table itself, but it seems to not support title tags!

```rst
+----------------------------+-----------------------------------------+
| Category                   | Apps                                    |
+============================+=========================================+
| 🎙 Audiobooks and Podcasts | |Acast Podcast Player| |AntennaPod|     |
|                            | |The Bob & Tom Show| |Castbox| |Google  |
|                            | Play Books|                             |
+-----------------+----------+-----------------------------------------+

.. |Acast Podcast Player| image:: https://play-lh.googleusercontent.com/QJ8PChzCg_lwdZLnsr_bliT_K88DMNi4UfWmK8zgjF7CgJ5mGzQZFV6ZrJUFiWJFfWS6=s30-rw
   :target: https://play.google.com/store/apps/details?id=com.acast.nativeapp
.. |AntennaPod| image:: https://play-lh.googleusercontent.com/VV5fdZWuZolh8Ii8FiuxD7LYCacmT76sCxyuMTm7kyfsHJkBWfkHSZNmqr1UGTo6JZCq=s30-rw
   :target: https://play.google.com/store/apps/details?id=de.danoeh.antennapod
.. |The Bob & Tom Show| image:: https://play-lh.googleusercontent.com/bwXFPAGsIBhp8xrFPATUzQk_y0pn230HHbMN9MxJ_GbTt5lN2-Fc_ShzyRtEfSDE9Vk=s30-rw
   :target: https://play.google.com/store/apps/details?id=com.radio.station.BOB.TOM
.. |Castbox| image:: https://play-lh.googleusercontent.com/CzFEOgmTdindoUqV4xDpZMknyjplmSUgOU-W9DtTuAZuyzZy_8fn26BZDRXzSwDbOTEh=s30-rw
   :target: https://play.google.com/store/apps/details?id=fm.castbox.audiobook.radio.podcast
.. |Google Play Books| image:: https://play-lh.googleusercontent.com/DglqS-eYHQYXnj8M8tmzh3JcKDXcidSo3IzgyCZzci8ZTV9Pmuk8vvIFh9XHOztC3Q=s30-rw
   :target: https://play.google.com/store/apps/details?id=com.google.android.apps.books```